.travis.yml

```diff
@@ -3,5 +3,17 @@ sudo: false
 go:
   - 1.5
   - 1.6
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
```

Comment:
Thanks for the PR, but this pulls in a 3rd party dependency for go dns. I don't think I want that (even for testing). What does this do?